services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/DeviceManagementService.java

```diff
@@ -129,6 +131,11 @@
      * @see <a href="https://www.eclipse.org/hono/docs/api/management/#/devices/searchDevicesForTenant">
      *      Device Registry Management API - Search devices for a tenant</a>
      */
-    Future<OperationResult<List<DeviceWithId>>> searchDevices(String tenantId, Optional<Integer> limit, Optional<Integer> offset,
-                                                              Optional<List<SortingOption>> sorting, Optional<List<Filter>> filters, Span span);
+    default Future<OperationResult<List<DeviceWithId>>> searchDevices(String tenantId, Optional<Integer> limit, Optional<Integer> offset,
+                                                              Optional<List<SortingOption>> sorting, Optional<List<Filter>> filters, Span span) {
+
```

Comment:
How about _null_ check of the arguments as described in the _JavaDoc_ of the `searchDevices` method? 
`@throws NullPointerException if any of the parameters is {@code null}.`

Reply:
ah good catch, thanks.